Automattic/sensei
# Add quiz pagination support for Learning Mode.

Fixes #4552
Fixes #4570

### Changes proposed in this Pull Request

* Introduces `sensei-lms/quiz-pagination` block that renders quiz pagination.

### Testing instructions


* Enable the feature flag: `add_filter( 'sensei_feature_flag_course_theme', '__return_true' );`
* Enable the feature flag: `add_filter( 'sensei_feature_flag_quiz_pagination', '__return_true' );`
* Enable Course Theme / Learning Mode for a course.
* Create a lesson with a quiz.
* Set pagination for the quiz.
* Visit the lesson page and take a quiz.
* Confirm the pagination works as expected.
* Make sure to test with different numbers per page. (1 question per page, 3 question per page...)


### Screenshot / Video


https://user-images.githubusercontent.com/2578542/148762086-06d45dc8-08a2-418e-869c-60bcc7c06f33.mp4
